### PR TITLE
Using spatie/laravel-permission v6

### PR DESCRIPTION
**Pull request by bldrvnlw**
Updated role and permissions from 5.8 to Laravel 10 and spatie/laravel-permission v6

### CODE REVIEW

**Comment by bldrvnlw** [comment]
I need to add a new version of seeder or perhave an artisan command for assiging the Admin role to one or more users to kickstart the administration.

File: routes/web.php
Comment (tasosmitsi) [inline routes/web.php:1]: 1. Routes from line 31:58 are now in an empty group. Is there any particular reason for that?
2. If you log in with a NON admin user and try to access localhost/admin/users from the browser directly you are still able to access the admin page. I believe adding isAdmin to the route middleware list for the admin page will solve the problem.
3. The same holds for the previously written admin page. We should add isAdmin to the route middleware list, that will hide the page.
4. You added a group of routes prefixed as admin. The thing is that there is a group with the exact same prefix (this group is used by the previously existing admin page). I need to test if those groups interfere with each other.

**Comment by tasosmitsi** [comment]
This commit https://github.com/vufuma/FUMA-webapp/pull/24/commits/146088d19b80e6a3577de7e6fbbd1a32e908d711
answers questions 2, and 3 of the previous comment.

File: routes/web.php
Comment (tasosmitsi) [reply routes/web.php:1]: As for q4:

Here is a list of the existing prefixed admin routes:

```
  GET|HEAD        admin ........................................................................ AdminController@index
  GET|HEAD        admin/analysis ........................................................ AdminController@showAnalysis
  GET|HEAD        admin/jobs ................................................................ AdminController@showJobs
  GET|HEAD        admin/permissions .................. permissions.index › RolesPermissions\PermissionController@index
  POST            admin/permissions .................. permissions.store › RolesPermissions\PermissionController@store
  GET|HEAD        admin/permissions/create ......... permissions.create › RolesPermissions\PermissionController@create
  GET|HEAD        admin/permissions/{permission} ....... permissions.show › RolesPermissions\PermissionController@show
  PUT|PATCH       admin/permissions/{permission} ... permissions.update › RolesPermissions\PermissionController@update
  DELETE          admin/permissions/{permission} . permissions.destroy › RolesPermissions\PermissionController@destroy
  GET|HEAD        admin/permissions/{permission}/edit .. permissions.edit › RolesPermissions\PermissionController@edit
  GET|HEAD        admin/roles .................................... roles.index › RolesPermissions\RoleController@index
  POST            admin/roles .................................... roles.store › RolesPermissions\RoleController@store
  GET|HEAD        admin/roles/create ........................... roles.create › RolesPermissions\RoleController@create
  GET|HEAD        admin/roles/{role} ............................... roles.show › RolesPermissions\RoleController@show
  PUT|PATCH       admin/roles/{role} ........................... roles.update › RolesPermissions\RoleController@update
  DELETE          admin/roles/{role} ......................... roles.destroy › RolesPermissions\RoleController@destroy
  GET|HEAD        admin/roles/{role}/edit .......................... roles.edit › RolesPermissions\RoleController@edit
  GET|HEAD        admin/search-jobs ............................................... AdvancedJobsSearchController@index
  POST            admin/search-jobs .............................................. AdvancedJobsSearchController@search
  POST            admin/search-jobs/action .............................. AdvancedJobsSearchController@containerAction
  GET|HEAD        admin/updates ............................................... updates.index › UpdateController@index
  POST            admin/updates ............................................... updates.store › UpdateController@store
  GET|HEAD        admin/updates/create ...................................... updates.create › UpdateController@create
  GET|HEAD        admin/updates/{update} ........................................ updates.show › UpdateController@show
  PUT|PATCH       admin/updates/{update} .................................... updates.update › UpdateController@update
  DELETE          admin/updates/{update} .................................. updates.destroy › UpdateController@destroy
  GET|HEAD        admin/updates/{update}/edit ................................... updates.edit › UpdateController@edit
  GET|HEAD        admin/users .................................... users.index › RolesPermissions\UserController@index
  POST            admin/users .................................... users.store › RolesPermissions\UserController@store
  GET|HEAD        admin/users/create ........................... users.create › RolesPermissions\UserController@create
  GET|HEAD        admin/users/{user} ............................... users.show › RolesPermissions\UserController@show
  PUT|PATCH       admin/users/{user} ........................... users.update › RolesPermissions\UserController@update
  DELETE          admin/users/{user} ......................... users.destroy › RolesPermissions\UserController@destroy
  GET|HEAD        admin/users/{user}/edit .......................... users.edit › RolesPermissions\UserController@edit
``` 
 As a result, there are no conflicts whatsoever regarding those routes. I consider these questions solved.

Comment (tasosmitsi) [reply routes/web.php:1]: This commit https://github.com/vufuma/FUMA-webapp/pull/24/commits/146088d19b80e6a3577de7e6fbbd1a32e908d711
answers questions 2, and 3 of the previous comment.

**Review by tasosmitsi — COMMENTED** [review]
I consider this review closed.

**Comment by bldrvnlw** [comment]
Regarding Q1: there isn't a good functional reason for this, the only justification is that it clearly puts all the non-auth top-level routes in a group together. But the same could also be achieved, and probably with a better explanation simply using comments. We could patch this in future.

**Pull request merged**